README.md

```diff
@@ -1,2 +1,85 @@
 # business-analytics
 Tools to produce Riff Business analytic reports on RiffEdu and RiffPlatform deployments
+
+This repository uses a third-party package called [Metabase][] to visualize usage metrics for Mattermost. Two docker containers are run; one using the metabase image and another using the
+official mysql image. Commands are provided to restore sql database backups from Mattermost to the mysql container.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Database Backups
+
+Download sql backups from the Riff Learning backup location (ask for the URL if you don't have it), and place all .sql files in the folder called 'database-backups' in the root of this repo.
```

Comment:
Added a note in the README under 'Getting Started' regarding which ports are used on your local machine by the docker containers.

Reply:
> I left some comments on practices.
> 
> I needed to do some extra work besides what was in the documentation to run the metabase server. Specifically, I needed to stop my own mysql server that was running locally.
> 
> I was also not able to test the functionality of the metabase server at this time because the username/password combinations are not widely known. It might be a better idea to put them in a secrets file that we distribute over slack, so that it is easier for devs and analysts to find them as needed?

Added a note in the README under 'Getting Started' regarding which ports are used on your local machine by the docker containers.